### PR TITLE
feat!: added entrypoint for user service startup 

### PR DESCRIPTION
**Pull request by v4l3rio**

### CODE REVIEW

File: entrypoint/src/main/kotlin/Main.kt [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import io.github.cdimascio.dotenv.dotenv
+import io.github.positionpal.AvroSerializer
+import io.grpc.Server
+import io.grpc.ServerBuilder
+import user.UserServiceImpl
+
+/**
+ * The port number for the gRPC server.
+ */
+const val PORT = 8080
```
Comment (giovaz94) [inline entrypoint/src/main/kotlin/Main.kt:13]: Hardcoded port on code.

File: entrypoint/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+plugins {
+    application
+}
+
+application {
+    mainClass.set("MainKt") // Sostituisci con la tua classe principale
```
Comment (giovaz94) [inline entrypoint/build.gradle.kts:6]: Remove comment

**Review by giovaz94 — COMMENTED** [review]
Good, green light after these two small refector

File: entrypoint/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    }
+    api(project(":storage"))
+    api(project(":grpc"))
+    api(project(":rabbitmq"))
+}
+
+tasks.named("run") {
+    dependsOn(":composeUp")
+    finalizedBy(":composeDown")
+}
```
Comment (tassiluca) [inline entrypoint/build.gradle.kts:25]: Why this is needed?

**Review by tassiluca — APPROVED** [review]

**Pull request merged**